src/lizaalert/courses/models.py

```diff
@@ -140,7 +140,13 @@ def current_lesson(self, user):
 
     @property
     def is_available(self):
-        """Проверить доступность курса."""
+        """
+        Проверить доступность курса.
+
+        Если дата начала курса не указана, то курс доступен всегда.
+        """
+        if not self.start_date:
+            return True
         return timezone.now().date() >= self.start_date
```

Comment:
лучше избегать условий c not (чтобы чтение кода было легче)


```suggestion
        if self.start_date:
            return timezone.now().date() >= self.start_date
        return True
```